### PR TITLE
app-admin/filebeat: version bump

### PR DESCRIPTION
**Pull request by hydrapolic**

### CODE REVIEW

**Comment by Dessa** [comment]
doing this would drop the only stable filebeat version for amd64, is this really desired?

**Comment by hydrapolic** [comment]
Yes you're right, I forgot about it, so let's just do a bump, I'll adjust the PR.

File: app-admin/filebeat/filebeat-6.1.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+SRC_URI="https://github.com/elastic/beats/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND=">=dev-lang/go-1.9.2"
+RDEPEND="!app-admin/filebeat-bin"
+
+RESTRICT="test"
```
Comment (SoapGentoo) [inline app-admin/filebeat/filebeat-6.1.1.ebuild]: `RESTRICT` after `KEYWORDS` please

File: app-admin/filebeat/filebeat-6.1.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+src_install() {
+	keepdir /var/{lib,log}/${PN}
+
+	fperms 0750 /var/{lib,log}/${PN}
+
+	newconfd "${FILESDIR}/${PN}.confd" ${PN}
+	newinitd "${FILESDIR}/${PN}.initd.1" ${PN}
+
+	insinto "/usr/share/doc/${PF}/examples"
+	doins ${PN}/{filebeat.yml,filebeat.reference.yml}
```
Comment (SoapGentoo) [inline app-admin/filebeat/filebeat-6.1.1.ebuild]:
```
docinto examples
dodoc ${PN}/{filebeat.yml,filebeat.reference.yml}
```

**Comment by hydrapolic** [comment]
Thanks you @SoapGentoo for the review, pushed an updated version.

**Comment by hydrapolic** [comment]
Thank you @mgorny

**Pull request closed without merging**